src/Nethermind/Nethermind.Blockchain/Synchronization/BeamSync/BeamSyncDb.cs

```diff
@@ -188,6 +190,11 @@ public bool KeyExists(byte[] key)
         }
 
         public IDb Innermost => _db.Innermost;
+        
+        public void Flush()
+        {
```

Comment:
what is this?

Reply:
This is for flushing DB manually. Works finally not used